de.metas.vertical.pharma.vendor.gateway.msv3/src/main/java/de/metas/vertical/pharma/vendor/gateway/msv3/interceptor/MSV3PharmaImportPartnerInterceptor.java

```diff
@@ -0,0 +1,98 @@
+/**
+ *
+ */
+package de.metas.vertical.pharma.vendor.gateway.msv3.interceptor;
+
+import org.adempiere.impexp.IImportInterceptor;
+import org.adempiere.impexp.IImportProcess;
+import org.adempiere.model.InterfaceWrapperHelper;
+import org.adempiere.util.Check;
+
+import de.metas.interfaces.I_C_BPartner;
+import de.metas.vertical.pharma.vendor.gateway.msv3.model.I_I_BPartner;
+import de.metas.vertical.pharma.vendor.gateway.msv3.model.I_MSV3_Vendor_Config;
+import lombok.NonNull;
+
+/*
+ * #%L
+ * metasfresh-pharma.vendor.gateway.msv3
+ * %%
+ * Copyright (C) 2017 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+/**
+ * @author metas-dev <dev@metasfresh.com>
+ *
+ */
+public class MSV3PharmaImportPartnerInterceptor implements IImportInterceptor
+{
+	public static final MSV3PharmaImportPartnerInterceptor instance = new MSV3PharmaImportPartnerInterceptor();
+
+	private static final String MSV3_Constant = "MSV3";
+
+	private static final String Password = "MSV3";
+
+	private static final String UserID = "MSV3";
```

Comment:
pls document what are those constants.
Why not having them as SysConfig with a fallback to there values?

Reply:
I will add documentation. These will be different for each record; We put some random value. But hose will be changed by the users.